api/src/main/java/org/pmiops/workbench/db/dao/UserDao.java

```diff
@@ -111,28 +111,37 @@ interface DbAdminTableUser {
 
     Timestamp getCreationTime();
 
+    @Deprecated // to be removed - use getDuccBypassTime
     Timestamp getDataUseAgreementBypassTime();
 
+    Timestamp getDuccBypassTime();
+
+    @Deprecated // to be removed
```

Comment:
I thought that the ticket was just removing references to Data Use Agreement. Why are these other functions being deprecated? Are they connected?

Reply:
No - I'm doing 2 things here.  These other columns were removed from the UI in https://github.com/all-of-us/workbench/pull/6629 so we could consider this a continuation of RW-7917.  I'll update the title.